src/linux/wpa-controller/include/Wpa/ProtocolHostapd.hxx

```diff
@@ -300,16 +300,42 @@ inline constexpr std::array<WpaKeyManagement, 27> AllWpaKeyManagements = {
 };
 
 /**
- * @brief All valid WpaKeyManagement values supporting fast-transition (FT).
+ * @brief A bitmask cotaining all valid WpaKeyManagement values supporting fast-transition (FT).
  */
-static constexpr std::underlying_type_t<WpaKeyManagement> WpaKeyManagementFt =
+static constexpr std::underlying_type_t<WpaKeyManagement> WpaKeyManagementMaskFt =
     std::to_underlying(WpaKeyManagement::Ieee80211x) |
     std::to_underlying(WpaKeyManagement::FtIeee8021x) |
     std::to_underlying(WpaKeyManagement::FtIeee8021xSha384) |
     std::to_underlying(WpaKeyManagement::FtSae) |
     std::to_underlying(WpaKeyManagement::FtFilsSha256) |
     std::to_underlying(WpaKeyManagement::FtFilsSha384);
 
+/**
+ * @brief All valid WpaKeyManagement values supporting fast-transition (FT).
+ */
+static constexpr std::initializer_list<WpaKeyManagement> WpaKeyManagementFt = {
```

Comment:
This is missing `FtPsk`

Reply:
The only one that we actually need right now ..

Reply:
Added